src/Novu/DTO/Topics/TopicCreateDto.cs

```diff
@@ -17,4 +17,7 @@ public class TopicCreateDto
     public string Name { get; set; }
 }
 
-public record RenameTopicRequest(string Name);
\ No newline at end of file
+public record RenameTopicRequest(string Name)
+{
+    public string Name { get; } = Name;
```

Comment:
Just curious what's the reason to have an explicit declaration for properties?

Reply:
however, this is not a very critical thing to have, just maybe a dev productivity improvement.